writers/gitignore-writer.js

```diff
@@ -0,0 +1,12 @@
+const { write } = require('../writers/writer');
+
+const createGitignore = path => {
+  const gitignore = 
+  `
+  node_modules
+`;
+
+  write(path + '../.gitignore', gitignore);
```

Comment:
on all of the writers, since you are giving it a path + filename here, you don't need the '..' in front of the slash, for example this should work:
```path + '/.gitignore'```